wiki/Tutorials/Loadbalancer/index.md

```diff
@@ -0,0 +1,161 @@
+# Documentation for setup of secure web-service with reverse-proxy and load balancer in OpenStack
+
+This guide will explain how to setup a web-service behind a reverse proxy. The reverse proxy will only be accessible over a load balancer from the internet. The web-service itself is secured in a separated network and only specific ports for incoming traffic are allowed for the web-service. The ports will be setup with security groups in OpenStack. Only one defined port for the forwarded requests from the reverse-proxy to the web-service and ssh for the configuration will be allowed. For the load balancer there will be a dmz network to secure the access from the load balancer to the reverse-proxy. A guide for the basic setup of a vm can be found in the quick start section of this wiki and will not be provided here.
```

Comment:
Please add the information, that the tutorial is written for de de.NBI Cloud site in Berlin in the beginning